### PR TITLE
DDOC-636 Add script for creating groups and collaborations 

### PR DESCRIPTION
**Pull request by bszwarc**
# Description

Added documentation for a script on creating / updating groups and collaborations.
Preview on staging: https://staging.developer.box.com/guides/cli/scripts/manage-groups-collaborations/

Fixes DDOC-636

### CODE REVIEW

File: content/guides/cli/scripts/manage-groups-collaborations.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+---
+# Manage groups and collaborations
+
+<!-- markdownlint-disable line-length -->
+
+## Script structure
+
+This script uses Box CLI to create or update groups, add users to them, and finally to create collaborations between groups and folders.
+The script consists of the following two parts. You can run them both or select one of them.
```
Comment (mgrytsai) [inline content/guides/cli/scripts/manage-groups-collaborations.md:25]: Suggest to define what two parts or just say "The script consists of the two parts"

File: content/guides/cli/scripts/manage-groups-collaborations.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+   |Group2| ManagedUser3@test.com|
+   |Group3| ManagedUser1@test.com|
+
+2. If the group doesn't exist, the script creates it. If it does exist, the script can update the entries based on the provided data.
+
+### Create or update collaborations
+
+1. The script uses the `.csv` file you specify for the
+   `CollaborationsCreationPath` parameter. The file lists group names, folder IDs, and collaboration roles. 
+2. For each row, the script checks if a group exists and if it's not already
```
Comment (mgrytsai) [inline content/guides/cli/scripts/manage-groups-collaborations.md:46]: Also, if group already exists, but CollaborationRole changed, script will update it if the parm `-UpdateExistingCollabs ` is passed

From the readme
> There may be cases where a group has already been a collaborator for a specific folder, but with a role other than that defined in the CSV file. This script will simply inform you about that case without making any change to existing collaboration. But if you want to update an existing collaboration with role defined in CSV file, you need to pass an additional switch -UpdateExistingCollabs when running the script.

**Review by mgrytsai — COMMENTED** [review]
Added 2 comments above.

File: content/guides/cli/scripts/manage-groups-collaborations.md (hunk 1 above)
Comment (bszwarc) [reply content/guides/cli/scripts/manage-groups-collaborations.md:25]: Sure, I will perhaps write that the parts are explained in the next sections.

File: content/guides/cli/scripts/manage-groups-collaborations.md (hunk 2 above)
Comment (bszwarc) [reply content/guides/cli/scripts/manage-groups-collaborations.md:46]: Added this bit to the explanation.

**Review by mgrytsai — APPROVED** [review]
LGTM

**Pull request merged**